Sources/MapboxCoreNavigation/EHorizon/RoadName.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import MapboxNavigationNative
+
+public struct RoadName {
```

Comment:
Could we use this struct outside EH context? There are some manipulations with road labeling so maybe we could refactor that later and benefit from this struct in multiple places?

Reply:
Related: https://github.com/mapbox/mapbox-navigation-ios/pull/2834#discussion_r585849144. It might even make sense for MapboxDirections to consolidate various RouteStep properties into a public [Road](https://github.com/mapbox/mapbox-directions-swift/blob/cec732768fd85429201e16fe9ece0dc15bc5e443/Sources/MapboxDirections/RouteStep.swift#L307) type that we could reuse here and elsewhere in MapboxCoreNavigation.